### PR TITLE
added performance checking module

### LINKED ISSUE #14: performance test (firebase)
need firebase performance test

### PR DESCRIPTION
**Pull request by inyukwo1**
Resolved #14 
added performance checking module for FBPixelManager.
In mac's android emulator, below was the result.
* ServerPerformanceTest/pixelWritePerformanceTest: Average Elapsed Time: 543.0ms, Average Updated Number: 2.0
* ServerPerformanceTest/bitmapReadPerformanceTest: Elapsed Time For getting 8x8 bitmap: 14ms
* ServerPerformanceTest/bitmapReadPerformanceTest: Elapsed Time For getting 16x16 bitmap: 64ms

### CODE REVIEW

File: app/src/main/java/com/canvearth/canvearth/server/FBPixelManager.java [hunk 1]
```diff
@@ -206,21 +211,27 @@ public void writePixelAsync(PixelData pixelData, Color color, @Nullable Runnable
...
         } catch (InterruptedException e) {
             Log.e(TAG, e.getMessage());
         }
+        return returnPixelData;
     }
 
-
-    private void updateParent(FBPixel childOriginPixel, FBPixel childNewPixel,
+    // returns last updated pixel
+    private PixelData updateParent(FBPixel childOriginPixel, FBPixel childNewPixel,
```
Comment (prodia4) [inline app/src/main/java/com/canvearth/canvearth/server/FBPixelManager.java]: Why does this need to return the last updated pixel?

Comment (inyukwo1) [reply app/src/main/java/com/canvearth/canvearth/server/FBPixelManager.java]: I used this in performance test, to see which number of update occurred.
Maybe client-side don't really have to use this. Then simply ignore this return value

Comment (prodia4) [reply app/src/main/java/com/canvearth/canvearth/server/FBPixelManager.java]: Ok

**Comment by prodia4** [comment]
Regarding the tests(I only checked the part in ServerInterfaceTest as this pull request has not been merged yet), They definitely pass, but when I bring the same logic to the app and actually run it, ANR happens.(When the UI thread of an Android app is blocked for too long, an "Application Not Responding" (ANR) error is triggered.) Could you check it please?

**Pull request merged**